client/test/e2e/shared/components.js

```diff
@@ -209,3 +209,47 @@ exports.findPatient = {
     submit.click();
   }
 };
+
+/**
+* Find Debtor Group component
+* findDebtorGroup.js
+* @public
+*/
+exports.findDebtorGroup = {
+  root     : element(by.css('[data-bh-find-debtorgroup-name]')),
+
+  search : function search(name) {
```

Comment:
Could you briefly describe each of these functions?  What is the difference between search and select?  Does one need to be called before the other?